src/EnergyPlus/PollutionModule.cc

```diff
@@ -275,1804 +276,1820 @@ void GetPollutionFactorInput(EnergyPlusData &state)
+                CheckFFSchedule(state,
+                                cCurrentModuleObject,
+                                "OtherFuel2",
+                                state.dataIPShortCut->cAlphaFieldNames(13),
+                                state.dataIPShortCut->cAlphaArgs(13),
+                                Pollution.OtherFuel2Coef.NMVOCSched,
+                                ErrorsFound);
+            }
+            Pollution.OtherFuel2Coef.Hg = state.dataIPShortCut->rNumericArgs(13);
+            if (!state.dataIPShortCut->lAlphaFieldBlanks(14)) {
+                CheckFFSchedule(state,
+                                cCurrentModuleObject,
+                                "OtherFuel2",
+                                state.dataIPShortCut->cAlphaFieldNames(14),
+                                state.dataIPShortCut->cAlphaArgs(14),
+                                Pollution.OtherFuel2Coef.HgSched,
+                                ErrorsFound);
+            }
+            Pollution.OtherFuel2Coef.Pb = state.dataIPShortCut->rNumericArgs(14);
+            if (!state.dataIPShortCut->lAlphaFieldBlanks(15)) {
+                CheckFFSchedule(state,
+                                cCurrentModuleObject,
+                                "OtherFuel2",
+                                state.dataIPShortCut->cAlphaFieldNames(15),
+                                state.dataIPShortCut->cAlphaArgs(15),
+                                Pollution.OtherFuel2Coef.PbSched,
+                                ErrorsFound);
+            }
+            Pollution.OtherFuel2Coef.Water = state.dataIPShortCut->rNumericArgs(15);
+            if (!state.dataIPShortCut->lAlphaFieldBlanks(16)) {
+                CheckFFSchedule(state,
+                                cCurrentModuleObject,
+                                "OtherFuel2",
+                                state.dataIPShortCut->cAlphaFieldNames(16),
+                                state.dataIPShortCut->cAlphaArgs(16),
+                                Pollution.OtherFuel2Coef.WaterSched,
+                                ErrorsFound);
+            }
+            Pollution.OtherFuel2Coef.NucHi = state.dataIPShortCut->rNumericArgs(16);
+            if (!state.dataIPShortCut->lAlphaFieldBlanks(17)) {
+                CheckFFSchedule(state,
+                                cCurrentModuleObject,
+                                "OtherFuel2",
+                                state.dataIPShortCut->cAlphaFieldNames(17),
+                                state.dataIPShortCut->cAlphaArgs(17),
+                                Pollution.OtherFuel2Coef.NucHiSched,
+                                ErrorsFound);
+            }
+            Pollution.OtherFuel2Coef.NucLo = state.dataIPShortCut->rNumericArgs(17);
+            if (!state.dataIPShortCut->lAlphaFieldBlanks(18)) {
+                CheckFFSchedule(state,
+                                cCurrentModuleObject,
+                                "OtherFuel2",
```

Comment:
I assume sometime the CheckFFSchedule function will be modified to take an enum as the third argument rather than the strings that are being used now.

Reply:
📌for now, I'll fix it soon.